mox/impl/Cargo.toml

```diff
@@ -18,7 +18,6 @@ proc-macro = true
 
 [dependencies]
 proc-macro2 = "1"
```

Comment:
Can you run `cargo ofl versions` once each for mox-impl and mox and bump the versions to `0.11.0`? I'll write a changelog section for you.

This will also require a patch version bump for any crates which are currently published and are changed by the above command. You can run the command once for each affected crate.

Reply:
Wow this is cool